cmd/argo/commands/lint.go

```diff
@@ -22,6 +22,10 @@ func NewLintCommand() *cobra.Command {
 		Use:   "lint FILE...",
 		Short: "validate files or directories of workflow manifests",
 		Run: func(cmd *cobra.Command, args []string) {
+			if len(args) == 0 {
```

Comment:
this actually intentional - works with xargs

Reply:
ok

Reply:
It feels a little unreasonable to just output "Workflow manifests validated" without adding any files.
```
$ argo lint
Workflow manifests validated
```

Reply:
kubectl use the `-f` parameter, If no file is specified, `pflag` will throw an error, and the output will print "flag needs an argument: 'f' in -f" and help information, exit code is 1.

`argo lint` director add file name, I think it is ok to directly output help information without specifying file, exit code is also 1. If need to be consistent with kubectl, we also need to use the `-f` parameter, which is a big change.

```
$ kubectl create -f
Error: flag needs an argument: 'f' in -f


Examples:
  # Create a pod using the data in pod.json.
  kubectl create -f ./pod.json
......



$ echo $?
1
```

Reply:
I'm not sure this is apples-and-apples. I guess the nearest would be `kubectl create --dry-run -f FILE`?

Either way, can you come up with a solution that works with `xargs`? We can't break existing users like this.

Reply:
ok